completions/bash/kpod

```diff
@@ -343,6 +343,24 @@ _kpod_ps() {
      _complete_ "$options_with_args" "$boolean_options"
 }
 
+_kpod_login() {
+     local options_with_args="
+     --username -u
+     --password -p
+     "
+     local boolean_options="
+     "
```

Comment:
-h --help

Reply:
Isn't --help, -h automatically added under `_kpod_kpod`?

Reply:
I think 
kpod -<tab> 
will show -h and --help

But that 
kpod login -<tab>  
Will not show -h and --help.

Reply:
oh ok, I will add it in.